### PR TITLE
Patch monorepo dependencies

### PR DESCRIPTION
**Pull request by ACatThatPrograms**
- Align tictactoe with mono repo dependencies to remove react and package duplicates for mono CI/CD process
 - Add alicenetjs dependencies and shims for browserify packages
 - Change eslint to WARN for unused declarations
 - Update TTT to use ReactDomV6

### CODE REVIEW

File: hw-tictac/src/components/proofOfConcept/ProofOfConcept.jsx [hunk 1]
```diff
@@ -2,26 +2,22 @@ import React from "react";
...
     return (
         <Box display="flex" flexDirection="column" gap={2}>
-            <Typography variant="h5">
-                AliceNet Tictactoe (POC)
-            </Typography>
+            <Typography variant="h5">AliceNet Tictactoe (POC)</Typography>
             <Box display="flex" flexDirection="row" gap={2}>
                 <Typography variant="span" fontFamily="Roboto">
-                    "This application is to demonstrate the use of multi-signature wallets where two wallets are coming
+                    This application is to demonstrate the use of multi-signature wallets where two wallets are coming
```
Comment (adrianbona) [inline hw-tictac/src/components/proofOfConcept/ProofOfConcept.jsx:10]: In my defense, I just copied whatever it was in the Prototype :P

File: hw-tictac/src/index.js [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-root.render(
-    <Provider store={store}>
-        <ThemeProvider theme={theme}>
-            <CssBaseline enableColorScheme />
-            <App />
-        </ThemeProvider>
-    </Provider>
-);
\ No newline at end of file
+ReactDOM.render(
```
Comment (adrianbona) [inline hw-tictac/src/index.js:10]: I just removed the warning by using the latest function provided by them. Is that a problem?

File: hw-tictac/src/components/header/HeaderMobile.jsx [hunk 3]
```diff
@@ -22,116 +13,106 @@ const WALLET_MAC_URL = process.env.REACT_APP_WALLET_MAC_URL;
...
-        },
-        {
-            label: "Windows",
-            location: WALLET_WINDOWS_URL,
-            icon: <FontAwesomeIcon icon={faWindows} />,
-            displayCallback: ({ location, label, icon }) =>
-                <MenuItem key={`wallet-section-${label}`} icon={icon} location={location} label={label} blank />
-        },
-    ];
+// const subSections =
```
Comment (adrianbona) [inline hw-tictac/src/components/header/HeaderMobile.jsx:47]: Considering we are using a versioning system, I would consider removing the code instead. It's less confusing at least to me.

**Review by adrianbona — APPROVED** [review]
Besides of the comments not related to this change's core, the rest LGTM

File: hw-tictac/src/components/proofOfConcept/ProofOfConcept.jsx (hunk 1 above)
Comment (ACatThatPrograms) [reply hw-tictac/src/components/proofOfConcept/ProofOfConcept.jsx:10]: Defense accepted. :joy_cat:

File: hw-tictac/src/index.js (hunk 2 above)
Comment (ACatThatPrograms) [reply hw-tictac/src/index.js:10]: Yeah, that should be all this is -- There was a collision on the versions I believe so I just mimicked what the other applications and the application boilerplate is using in the monorepo

File: hw-tictac/src/components/header/HeaderMobile.jsx (hunk 3 above)
Comment (ACatThatPrograms) [reply hw-tictac/src/components/header/HeaderMobile.jsx:47]: Sounds good to me , wasn't sure if you needed it -- Will remove

**Review by adrianbona — APPROVED** [review]
LGTM

**Pull request merged**